internal/go_repository_test.go

```diff
@@ -75,3 +81,56 @@ func TestDirectives(t *testing.T) {
 		t.Fatal("Unexpected error:\n", err)
 	}
 }
+
+func TestRepoConfig(t *testing.T) {
+	if err := bazel_testing.RunBazel("build", "@bazel_gazelle_go_repository_config//:all"); err != nil {
+		t.Fatal(err)
+	}
+	outputBase, err := getBazelOutputBase()
+	if err != nil {
+		t.Fatal(err)
+	}
+	outDir := outputBase + "/external/bazel_gazelle_go_repository_config"
```

Comment:
Use `filepath.Join` for Windows compatibility instead of string concatenation with `/` characters.

I think this test is disabled on Windows because it hits the command line length limit for some reason. Better to keep it working to the extent possible though.